db/seeds.rb

```diff
@@ -17,3 +17,9 @@
 Diversities::SEED_DISABILITIES.each do |disability|
   Disability.find_or_create_by!(name: disability.name, description: disability.description)
 end
+
+Dttp::CodeSets::SubjectSpecialisms::MAPPING.keys.each do |subject_specialism|
+  AllocationSubject.find_or_create_by(name: subject_specialism[:allocation_subject])
+  SubjectSpecialism.find_or_create_by(name: subject_specialism[:name], marketing_name: subject_specialism[:marketing_name],
+                                      allocation_subject_id: AllocationSubject.find_by(name: subject_specialism[:allocation_subject]).id)
```

Comment:
Preferable to use raise exceptions instead of failing silently
```suggestion
  allocation_subject = AllocationSubject.find_or_create_by!(name: subject_specialism[:allocation_subject])
  allocation_subject.subject_specialisms.find_or_create_by!(name: subject_specialism[:name], marketing_name: subject_specialism[:marketing_name])
```